docker/benchmark-builder/Dockerfile

```diff
@@ -34,6 +34,9 @@ ARG benchmark
 # Set up source and output directories.
 ENV SRC /src
 ENV OUT /out
+
```

Comment:
Won't this be inconsistent with ossfuzz?

Reply:
For the record I'm designing a bit of a refactor of docker images so that ossfuzz images will use this. Should get started later today on this.

Reply:
Yes i changed oss-fuzz-builder/Dockerfile as well in this CL.

Reply:
Didn't see, sorry